features/datalist.yml

```
@@ -0,0 +1,24 @@
+name: <datalist>
+description: The `<datalist>` element with `<options>` child elements defines a set of recommended values for an `<input>` element, connected by a `list` attribute set to the ID of the `<datalist>`.
```

@ddbeck how about this simplification and elaboration:

```suggestion
description: The `<datalist>` element defines a set of recommended values for an `<input>` element. Browsers may show a dropdown menu of all options, or matching options as the user types.
```